compiler/modules/front/front.bal

```diff
@@ -60,7 +60,9 @@ class Module {
 
 public function loadModule(string filename, bir:ModuleId id) returns bir:Module|err:Any|io:Error {
     string contents = check io:fileReadString(filename);
-    ModulePart part = check parseModulePart(contents);
+    Tokenizer tok = new (contents);
+    check tok.advance();
```

Comment:
This change is not OK. This exposes front.bal to Tokenizer, whereas now it's encapsulated in the parser.